### ISSUE TITLE
D_loss drop fast and G_loss drop slow

### ISSUE THREAD

**Issue by KirtoXX** [junyanz/pytorch-CycleGAN-and-pix2pix#858]
When I train my dataset,I fand that loss of Discriminative drop very fast,after 3 epoch ,loss_d is about 0.001,but the l1 loss is about 0.18 and the result of G is bad.

**Comment by junyanz** [comment]
You can consider using a G with a higher capacity.

**Comment by 13462877152** [comment]
> You can consider using a G with a higher capacity.

Hello, Professor, Thank you for your reply.  I am puzzled about this issue. Why the loss of D drop very fast , we should using a G with higher capacity ?   maybe  because  the result of G is bad ?  Is it  possible to use a reduced Discriminator?

**Comment by junyanz** [comment]
You can also try a reduced D. But sometimes a reduced D cannot capture the difference between real and fake samples.

**Comment by 13462877152** [comment]
Thank you very much for your help.  I will try the reduced D in the project .

**Comment by Rjaat** [comment]
> You can also try a reduced D. But sometimes a reduced D cannot capture the difference between real and fake samples.

Hi @junyanz can you explain a bit about reduced D and higher capacity G please!! like how it can be achieved?
